plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java

```diff
@@ -571,7 +571,7 @@ private <T> T executeWithSession(SessionCallable<T> sessionCallable)
                     throw e;
                 }
                 long delay = Math.min(schedule.nextDelay().toMillis(), timeLeft);
-                log.warn(e.getMessage());
```

Comment:
per https://github.com/trinodb/trino/pull/14933#discussion_r1095750818 i don't think we should do a change here

Reply:
Reverted